### ISSUE TITLE
something confusing in calculating the loss.

### ISSUE THREAD

**Issue by SiyuWang15** [rosinality/glow-pytorch#24]
In the paper, when x is continuous, the loss is calculated by feeding  $x+U(0,a)$ into the model, but in the codes, it seems you feed $x+N(0,a)$ into the model?

**Comment by rosinality** [comment]
Do you mean this line? https://github.com/rosinality/glow-pytorch/blob/97081ff115a694cf04aeedbd58447f33d242c879/train.py#L127 It injects uniform noise.

**Comment by SiyuWang15** [comment]
Oh, sorry, I made a stupid mistake and you are right. Thank you for your reply.
